File: src/main/java/frc/team4069/robot/commands/elevator/StopElevatorCommand.java
package frc.team4069.robot.commands.elevator;

import frc.team4069.robot.commands.CommandBase;

public class StopElevatorCommand extends CommandBase {

    public StopElevatorCommand() {
        requires(elevator);
    }

    @Override
    protected void initialize() {
        elevator.stop();
    }

    @Override
    protected boolean isFinished() {
        return true;
    }
}

File: src/main/java/frc/team4069/robot/commands/elevator/StopElevatorCommand.kt
package frc.team4069.robot.commands.elevator

import edu.wpi.first.wpilibj.command.InstantCommand
import frc.team4069.robot.subsystems.ElevatorSubsystem

// Command to stop the elevator
class StopElevatorCommand : InstantCommand() {

    init {
        requires(ElevatorSubsystem)
    }

    override fun initialize() {
        ElevatorSubsystem.stop()
    }
}